<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>751</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="encryptTab">
     <attribute name="title">
      <string>Encryption</string>
     </attribute>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="encryptBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>371</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt the File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>641</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="serveBtn">
      <property name="geometry">
       <rect>
        <x>394</x>
        <y>300</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Serve on Tor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Public Key:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="pub_key">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>731</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="decryptTab">
     <attribute name="title">
      <string>Decrypt</string>
     </attribute>
     <widget class="QPushButton" name="browseBtn_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Private Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse File:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="encryptBtn_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>731</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt the File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>641</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="pub_key_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>731</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>731</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
